py3status/modules/github.py

```diff
@@ -4,7 +4,8 @@
 
 To check notifications a Github `username` and `personal access token` are
 required.  You can create a personal access token at
-https://github.com/settings/tokens The only `scope` needed is `notifications`,
+https://github.com/settings/tokens/new?scopes=notifications&description=py3status-query-github
+The only `scope` needed is `notifications` is selected automatically for you,
```

Comment:
Greetings, `First-time contributor`! :wave: 

Can you rename the description to just `py3status`?

And we don't work with C/C++ here. Maybe you wanted to try this at `conky`? :blush:

Reply:
My bad, too much C/C++ in my head :rofl: